src/Utilities/Compiler/Options/EmptyCategorizedAnalyzerConfigOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Analyzer.Utilities
+{
+    internal sealed class EmptyCategorizedAnalyzerConfigOptions : ICategorizedAnalyzerConfigOptions
```

Comment:
Is this a new type or existing type being moved to a separate file?

Reply:
@mavasani New :(

Reply:
Actually I think I introduced it, used it, then removed usage in a later commit. Will double check and delete it.